A change in Qt 5.11_beta3 leads to errors

Arch GNU/Linux
```
CMake Error at src/watchdog/CMakeLists.txt:8 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```
```
CMake Error at src/libqtdbustest/CMakeLists.txt:25 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```
```
CMake Error at src/qdbus-test-runner/CMakeLists.txt:12 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```
```
CMake Error at tests/CMakeLists.txt:27 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

According to [the official Qt documentation](https://doc.qt.io/qt-5/cmake-manual.html) `qt5_use_modules` macro is obsolete

--- a/src/libqtdbustest/CMakeLists.txt
+++ b/src/libqtdbustest/CMakeLists.txt
@@ -22,11 +22,11 @@ add_library(
 	${QTDBUSTEST_SRC}
 )
 
-qt5_use_modules(
+target_link_libraries(
 	qtdbustest
-	Core
-	DBus
-	Test
+	Qt5::Core
+	Qt5::DBus
+	Qt5::Test
 )
 
 set_target_properties(

--- a/src/qdbus-test-runner/CMakeLists.txt
+++ b/src/qdbus-test-runner/CMakeLists.txt
@@ -9,9 +9,9 @@ target_link_libraries(
 	qtdbustest
 )
 
-qt5_use_modules(
+target_link_libraries(
 	qdbus-simple-test-runner
-	Core
+	Qt5::Core
 )
 
 install(

--- a/src/watchdog/CMakeLists.txt
+++ b/src/watchdog/CMakeLists.txt
@@ -5,9 +5,9 @@ add_executable(
 	main.cpp
 )
 
-qt5_use_modules(
+target_link_libraries(
 	watchdog
-	Core
+	Qt5::Core
 )
 
 install(

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -24,11 +24,11 @@ add_executable(
 	${UNIT_TESTS_SRC}
 )
 
-qt5_use_modules(
+target_link_libraries(
 	unit-tests
-	Core
-	DBus
-	Test
+	Qt5::Core
+	Qt5::DBus
+	Qt5::Test
 )
 
 target_link_libraries(

